src/services/conversationsService.js

```diff
@@ -324,9 +343,26 @@ const changeReadOnlyState = async function(token, readOnly) {
 	}
 }
 
+/**
+ * Change the listable scope
+ * @param {string} token The token of the conversation to be modified
+ * @param {int} listable The new listable scope to set
+ */
+const changeListable = async function(token, listable) {
+	try {
+		const response = await axios.put(generateOcsUrl('apps/spreed/api/v3', 2) + `room/${token}/listable`, {
+			scope: listable,
+		})
+		return response
+	} catch (error) {
+		console.debug('Error while updating listable scope: ', error)
```

Comment:
Since you catch here, `await this.$store.dispatch('setListable', {` doesn't catch anything anymore and does not `showError`, but always shows the success message.
can be tested by putting
```php
return new DataResponse([], Http::STATUS_BAD_REQUEST);
```
In the first line of the function in the RoomController.

From my pov the services should never catch.